src/Agent.Worker/SignatureService.cs

```diff
@@ -30,14 +30,20 @@ public async Task<Boolean> VerifyAsync(Definition definition, CancellationToken
 
             var configurationStore = HostContext.GetService<IConfigurationStore>();
             AgentSettings settings = configurationStore.GetSettings();
-            String fingerprint = settings.Fingerprint;
+            SignatureVerificationSettings verificationSettings = settings.SignatureVerification;
             String taskZipPath = definition.ZipPath;
             String taskNugetPath = definition.ZipPath.Replace(".zip", ".nupkg");
 
             // Rename .zip to .nupkg
             File.Move(taskZipPath, taskNugetPath);
 
-            String arguments = $"verify -Signatures \"{taskNugetPath}\" -CertificateFingerprint {fingerprint} -Verbosity Detailed";
```

Comment:
I think we lose back compat with this. Skype is using `Fingerprint` so we need to keep that around until we roll out the Agent, and they stop using that feature in place of the new one. In the interim we need to keep it.

Reply:
It is still around. There is still a Fingerprint setter on AgentSettings that creates the new structure for backward compatibility. Also the SignatureVerification_FailsWhenTasksArentSignedWithFingerprint L1 test still exercises that codepath.